test/e2e/lib/driver-helper.js

```diff
@@ -272,6 +272,12 @@ export function setWhenSettable(
 				return null;
 			}
 			await highlightElement( driver, element );
+			const currentValue = await element.getAttribute( 'value' );
+			if ( currentValue ) {
+				for ( let i = 0; i < currentValue.length; i++ ) {
+					await element.sendKeys( Key.BACK_SPACE );
```

Comment:
I'd normally use the [element.clear()](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_WebElement.html#clear) here, but it has some [weird side effects](https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/214) that make i.e. the main block inserter to close when called on the search input. Sending keys seems to work consistently.